.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on: [pull_request, pull_request_review]
```

Comment:
Make the workflow only triggered on pull req main

Reply:
Maybe only impl that trigger on push main

Reply:
Maybe do pull_request_review trigger only on CI workflow and on CI complete for CD workflow.

Reply:
Ok